### PR TITLE
RLM-1438 Ensure we're using openstack_hosts fork

### PR DESCRIPTION
**Pull request by antonym**
Injects the openstack_hosts fork on 14.7.0 which
includes the domain fixes for RLM-322.

### CODE REVIEW

**Comment by antonym** [comment]
recheck_cit_trusty_aio_swift_liberty_to_newton_leap

**Comment by antonym** [comment]
recheck_cit_trusty_aio_swift_kilo_to_newton_leap

File: scripts/pre_redeploy.sh [hunk 1]
```diff
@@ -124,3 +124,10 @@ pushd /etc/openstack_deploy/
     sed -i '/^haproxy_package_state.*/d' user_osa_variables_defaults.yml
     echo "haproxy_package_state: 'latest'" >> user_osa_variables_defaults.yml
 popd
+
+# RLM-1438 patch openstack_hosts fork into 14.7.0 to ensure RLM-322 works
+if [[ ${RPC_TARGET_CHECKOUT} == "r14.7.0" ]]; then
```
Comment (Frank-ZhangXin) [inline scripts/pre_redeploy.sh]: Do we care about any version lower than 14.7.0?

Comment (antonym) [reply scripts/pre_redeploy.sh]: Don't think we're deploying anything older via upgrades and this should be taken care of in 14.8.0.

**Comment by claco** [comment]
Thanks @BjoernT !

**Pull request merged**